src/onboarding2/Welcome/Welcome.js

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback, useMemo } from 'react'
+import PropTypes from 'prop-types'
+import { ButtonText, DropDown, GU, Layout, Split, useTheme } from '@aragon/ui'
+import { network } from '../../environment'
+import Header from '../Header/Header'
+import OpenOrg from './OpenOrg'
+import Suggestions from './Suggestions'
+import WelcomeAction from './WelcomeAction'
+
+import actionCreate from './assets/action-create.png'
+import actionOpen from './assets/action-open.png'
+
+function Welcome({ onBack, onOpen, onCreate, openMode, selectorNetworks }) {
+  const theme = useTheme()
+
+  const selectorNetworksSorted = useMemo(() => {
+    return selectorNetworks
+      .map(([type, name, url]) => ({ type, name, url }))
+      .sort((a, b) => {
+        if (b.type === network.type) return 1
+        if (a.type === network.type) return -1
+        return 0
+      })
+  }, [selectorNetworks])
+
+  const changeNetwork = useCallback(
+    index => {
+      window.location = selectorNetworksSorted[index].url
+    },
+    [selectorNetworksSorted]
+  )
+
+  return (
+    <div
+      css={`
+        display: grid;
+        align-items: center;
+        justify-content: center;
+      `}
+    >
+      <Layout
+        breakpoints={{
+          medium: 84 * GU,
+          large: 112 * GU,
+        }}
+      >
+        <Header
+          title="Welcome to Aragon"
+          subtitle="Create your own organization and token in a few minutes!"
+        />
+
+        <Split
+          primary={
+            openMode ? (
+              <OpenOrg onBack={onBack} />
+            ) : (
+              <div>
+                <DropDown
+                  items={selectorNetworksSorted.map(network => network.name)}
+                  placeholder={selectorNetworksSorted[0].name}
+                  onChange={changeNetwork}
+                  wide
+                />
+                <WelcomeAction
```

Comment:
![Image-1](https://user-images.githubusercontent.com/3072458/64344370-9c1fb300-cfee-11e9-90fc-a4d2a44a24ac.jpg)

Got this rendering of the images using Metamask in iOS.